source/adapters/opencl/enqueue.cpp

```diff
@@ -409,21 +403,21 @@ UR_APIEXPORT ur_result_t UR_APICALL urEnqueueReadHostPipe(
   }
 
   cl_ext::clEnqueueReadHostPipeINTEL_fn FuncPtr = nullptr;
-  ur_result_t RetVal =
+  UR_RETURN_ON_FAILURE(
       cl_ext::getExtFuncFromContext<cl_ext::clEnqueueReadHostPipeINTEL_fn>(
           CLContext, cl_ext::ExtFuncPtrCache->clEnqueueReadHostPipeINTELCache,
-          cl_ext::EnqueueReadHostPipeName, &FuncPtr);
+          cl_ext::EnqueueReadHostPipeName, &FuncPtr));
 
   if (FuncPtr) {
-    RetVal = mapCLErrorToUR(
+    CL_RETURN_ON_FAILURE(mapCLErrorToUR(
```

Comment:
```suggestion
    CL_RETURN_ON_FAILURE(
```
`CL_RETURN_ON_FAILURE` does `mapCLErrorToUR` for you (same [here](https://github.com/oneapi-src/unified-runtime/pull/1448/files#diff-c80bbd3e96dbcb2b2ca1d99af816a310b1720decee46fdb0e97d5fd3075e5cb3R444))